tools/devtool

```diff
@@ -722,7 +722,12 @@ cmd_test() {
     ensure_devctr
     ensure_build_dir
     ensure_ci_artifacts
-    [ $do_build != 0 ] && cmd_build --release
+    if [ $do_build != 0 ]; then
+      cmd_build --release
+      if [ -n "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" ]; then
```

Comment:
It sounds a bit awkward that we use an environment variable called `BUILDKITE_PULL_REQUEST_BASE_BRANCH` for an A/B test that runs outside of the CI. It's really more of a nit, but could we find a more intuitive name for this?